build/rust/BUILD.gn

```diff
@@ -31,6 +35,12 @@ brave_ffi_deps = [
 
 brave_rust_deps = brave_ffi_deps + [ ":rust_lib" ]
 
+if (is_win) {
```

Comment:
you are adding to brave_ffi_deps after it gets used ini brave_rust_deps

Reply:
also this should be `enable_rpill`

Reply:
Not using rust anymore.